lsp-mode.el

```diff
@@ -7614,7 +7614,7 @@ When prefix UPDATE? is t force installation even if the server is present."
 
 ;;;###autoload
 (defun lsp-update-server (&optional server-id)
-  "Interactively update a server."
+  "Interactively update (reinstall) a server."
```

Comment:
`lsp-update-server` and `lsp-update-servers` reinstall all servers, right? Even if already up-to-date?

Reply:
yes, ATM we don't have a mechanism to check if it is up to date.